rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -707,6 +707,12 @@ object Reduce {
                                  STRING_APPEND_COST * (lhs.length + rhs.length)
                                )
                          } yield Expr(GString(lhs + rhs))
+                       case (GByteArray(lhs), GByteArray(rhs)) =>
+                         for {
+                           _ <- costAccountingAlg.charge(
+                                 STRING_APPEND_COST * (lhs.size() + rhs.size())
```

Comment:
I think that the cost for concatenating to byte arrays should what the cost for concatenating lists is, rather than strings. Please change to:
```scala
costAccountingAlg.charge(PREPEND_COST * lhs.size())
```
and we will adjust if this is not the case. In general, it is more consistent as byte array and list are both collections and `String` is not.